src/App.js

```diff
@@ -1,19 +1,28 @@
 import React, { Component } from 'react';
+import {observer} from "mobx-react";
 import logo from './logo.svg';
 import './App.css';
 
+import Users from './models/Users';
```

Comment:
Try to not import things, that you are not going to use in this file.
I really think, we need to check ESLint configuration :D